src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.js

```diff
@@ -60,7 +60,7 @@ const ImageRenderer = (props) => {
                 >
                     {({show}) => (
                         <PressableWithoutFocus
-                            style={styles.noOutline}
+                            styles={styles.noOutline}
```

Comment:
Also, there was a typo here since the name of prop for `PressableWithoutFocus` is `styles` not `style`. 

https://github.com/Expensify/App/blob/213ba5509a51a812bed795d182d65da01f1d13e8/src/components/PressableWithoutFocus.js#L17

I noticed this while reviewing the PR for #19717 so I am fixing it here.

Reply:
I get this console error:
<img width="338" alt="image" src="https://github.com/Expensify/App/assets/50919443/e2883b3c-ac31-43f1-9b7b-f71dd05c426b">

Reply:
@parasharrajat can you link the PR if possible? 
Because if that’s a big one and will take time to be merged, I can raise a PR and fix this sooner.

Reply:
@mollfpr @cristipaval 
`styles` has to be an array here and I passed an object.

I made this typo while fixing an issue created in a related PR. Shall I create a new PR for this so that it is fixed before QA testing? Otherwise if the other PR that @parasharrajat is talking about takes a lot of time then this PR will be blocked unnecessarily.